app/Managers/PostManager.php

```diff
@@ -135,7 +135,7 @@ public function destroy($postId)
         $trashed = $this->repository->destroy($postId);
 
         if ($trashed) {
-            $this->fastly->purgeKey('post-'.$postId);
+            // $this->fastly->purgeKey('post-'.$postId);
```

Comment:
This probably shouldn't be commented, right?

Reply:
oops yes - commented out for testing on local ! Thanks for catching that! Commented back in :)